backend/src/main/java/com/senior/project/backend/submissions/SubmissionRepository.java

```diff
@@ -21,4 +21,10 @@ List<Submission> findAllBeforeNowWithUserAndTask(
         @Param("userId") UUID userId, 
         @Param("taskId") int taskId
     );
+
+    @Query("SELECT s FROM Submission s WHERE s.submissionDate < :now AND s.studentId = :userId")
```

Comment:
Why do these filter based on time? In what scenario is the submission date in the future?

Reply:
not sure, this was jimmy's code